src/aiida/orm/utils/mixins.py

```diff
@@ -181,22 +183,21 @@ class Sealable:
 
     SEALED_KEY = 'sealed'
 
-    __qb_fields__ = [
-        add_field(
-            SEALED_KEY,
-            dtype=bool,
-            doc='Whether the node is sealed',
-        ),
-    ]
+    class Model(pydantic.BaseModel):
+        sealed: bool = MetadataField(description='Whether the node is sealed')
 
     @classproperty
     def _updatable_attributes(cls) -> tuple[str, ...]:  # noqa: N805
         return (cls.SEALED_KEY,)
 
+    @property
+    def sealed(self) -> bool:
+        return self.base.attributes.get(self.SEALED_KEY, False)  # type: ignore[attr-defined]
+
     @property
     def is_sealed(self) -> bool:
         """Returns whether the node is sealed, i.e. whether the sealed attribute has been set to True."""
-        return self.base.attributes.get(self.SEALED_KEY, False)  # type: ignore[attr-defined]
+        return self.sealed
```

Comment:
Keeping both for backwards compatibility, or is there a different need?

Reply:
Yes, keeping it for backwards compatibility. In the model, I define the field as `sealed` so the (de)serialization code expects a `sealed` property, which is why I add it. I could have also chosen to not add the property and just name the field `is_sealed` instead. Or, we could keep `sealed` as the field and then define the `model_to_orm` attribute as `model_to_orm: lambda model: model.is_sealed`.

Reply:
I think this is fine. Quickly checking other ORM model-to-class-methods standard suggests you're consistent, so let's keep it as is.